tmt/frameworks/__init__.py

```diff
@@ -101,18 +102,23 @@ def get_test_command(
     def get_pull_options(
         cls,
         invocation: 'TestInvocation',
+        existing_options: Optional[TransferOptions],
```

Comment:
I'd probably pick slightly different name, those options exist in any case, maybe "current" options? Or just "options"?